docs/user/security/api-keys/index.asciidoc

```diff
@@ -0,0 +1,85 @@
+[role="xpack"]
+[[api-keys]]
+=== API Keys
+
+An API key enables you to create secondary credentials with the same or lower 
```

Comment:
How about:

`API keys enable you to create secondary credentials so that you can send requests on behalf of other Kibana users. Secondary credentials have the same or lower access rights.`